src/cpp/session/modules/SessionJobs.R

```diff
@@ -78,3 +78,14 @@
    invisible(NULL)
 })
 
+.rs.addApiFunction("runScriptJob", function(path, workingDir = NULL, 
+                                            importEnv = FALSE,
```

Comment:
Is it intentional that `importEnv` and `exportEnv` are not passed through to the `.Call()` bit below?